src/main/java/com/hillel/shabrin/homeworks/homework7/Main.java

```diff
@@ -8,50 +8,53 @@ public class Main {
     public static void main(String[] args) {
 
         //pt 6
-        String[] words = {"apple", "orange", "lemon", "banana",
-                "apricot", "avocado" , "broccoli", "carrot",
-                "cherry", "garlic", "grape", "melon", "leak",
-                "kiwi", "mango", "mushroom", "nut", "olive",
-                " pea", "peanut", "pear", "pepper", "pineapple",
-                "pumpkin", "potato"};
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" , "broccoli",
+                "carrot", "cherry", "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom",
+                "nut", "olive", "pea", "peanut", "pear", "pepper", "pineapple", "pumpkin", "potato"};
+
+        Random random = new Random();
+
+        String word = words[random.nextInt(words.length)];
+
+        int attempts = 0;
+
+        StringBuilder guess = new StringBuilder();
+        for (int i = 0; i < word.length(); i++) {
+            guess.append("#");
+        }
 
         Scanner scanner = new Scanner(System.in);
-        System.out.println("Hello! You need to guess one word that I thought of.");
-        System.out.println("This word refers to the theme of vegetables and fruits.");
 
-        Random random = new Random();
+        while (true) {
 
-        String answer = words[random.nextInt(words.length)];
-        char[] answerChars = answer.toCharArray();
-        char[] guessChars = new char[answerChars.length];
-        Arrays.fill(guessChars, '#');
+            System.out.println("Guess the word: " + guess);
 
-        boolean guessed = false;
-        while (!guessed) {
-            System.out.print("Guess the word: ");
-            String guess = scanner.nextLine().toLowerCase();
+            System.out.print("Enter your guess (lowercase only): ");
+            String answer = scanner.nextLine();
 
-            if (guess.equals(answer)) {
-                guessed = true;
+            if (answer.equals(word)) {
+                System.out.println("Congratulations! You guessed the word in " + attempts + " attempts.");
+                break;
             } else {
```

Comment:
if you have break in `if`, you dont need `else` here

Reply:
yeah, I know that. just gotta get used to it. thanks :)